habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -150,14 +150,20 @@ class Eq2CubeConfig(ObsTransformConfig):
     )
 
 
+@dataclass
+class TmpObsTransformConfig(HabitatBaselinesBaseConfig):
+    # TODO : Make this a dict of str:ObsTransformConfig rather than use an enabled_transforms list
+    enabled_transforms: List[str] = field(default_factory=lambda: [])
```

Comment:
Currently a hack. Needs to change later. This is to get around the `enabled_transforms` which will not exist in the future.